matplotlib-setup.cfg

```diff
@@ -0,0 +1,43 @@
+# Rename this file to setup.cfg to modify Matplotlib's build options.
```

Comment:
is this needed?

Reply:
I remove the command, I just copied the file and have overseen this command. It works without renaming, because it is defined in an environment variable